### ISSUE TITLE
scroom does not open 16bpp TIF images, and exits without reporting it in the GUI

### ISSUE THREAD

**Issue by JoostvanPinxten** [kees-jan/scroom#76]
The attached image does not open with scroom, and it does not show a popup with the error.

On the command line, I noticed that it said that 16 bps TIF is not supported. 

[blob.zip](https://github.com/kees-jan/scroom/files/5213270/blob.zip)

**Comment by kees-jan** [comment]
Do you need 16bps support? Otherwise, this is a duplicate of #74

**Comment by JoostvanPinxten** [comment]
Yes, as it turns out, we do need 16bps support. But viewing it at 8bps is sufficient. I guess throwing away the 8 LSB will be enough. We do not nees the pipet support for this yet. Will see whether I can make a pull request for this sometime.

**Comment by kees-jan** [comment]
Error reporting has been much improved, but you won't be able to see it, since I've also added support for 16-bit tifs. Including Pipette 😉
